uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -698,4 +723,17 @@ private MqttProtoVersion convertMqttVersion(String mqttVersion) {
 
         return version;
     }
+
+
+    @Data
+    @AllArgsConstructor
+    private class MqttBrokerConnection {
+
+        private String host;
+        private Integer port;
+        private List<String> caList;
```

Comment:
That store CAs multiple times.

Actually CAs is a property of group as well as list of addresses.

Reply:
This class was created to make connect to broker logic independent from IoT discovery classes usage (GGGroup, GGCore etc) because we need ability to connect to other brokers without prior discovery.
Regarding to data storage: 
List<String> caList = group.getCAs();
caList stores pointer to original collection, not the copy